pkg/metrics/server.go

```diff
@@ -0,0 +1,120 @@
+package metrics
```

Comment:
No need to add this file.

Please refer `NewHealthzAndMetricsHandler` to learn more.